build_scripts/performance_test.py

```diff
@@ -36,10 +37,12 @@ def run_performance_test(time_perf):
         print("input file is not generated")
         sys.exit(1)
     time0 = time.time()
-    call_subprocess('python3 -m similarity_processor.similarity_cmd --p "%s" --u 0 --c "1"' % input_file)
+    call_subprocess('python3 -m similarity --p "%s"'
```

Comment:
Why python3? Is this because of the docker-images used for the build? 

If yes, can we consider docker images from: https://hub.docker.com/_/python

Maybe worth considering docker image size between ubuntu and alpine, and if this is only for our testing, we can save some time by choosing an image that's of a lesser size? 

Also - if the image comes with default python3, we can then change this command to a more generic one like: python -m similarity --p "%s" ?

Reply:
Agree with generic python call. accepted for refactoring. Currently the action script is using the ubuntu VM, not docker based.